packages/react-components/react-breadcrumb/src/components/BreadcrumbLink/useBreadcrumbLinkStyles.ts

```diff
@@ -1,33 +1,80 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { BreadcrumbLinkSlots, BreadcrumbLinkState } from './BreadcrumbLink.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
 
 export const breadcrumbLinkClassNames: SlotClassNames<BreadcrumbLinkSlots> = {
   root: 'fui-BreadcrumbLink',
-  // TODO: add class names for all slots on BreadcrumbLinkSlots.
-  // Should be of the form `<slotName>: 'fui-BreadcrumbLink__<slotName>`
+  icon: 'fui-BreadcrumbLink__icon',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'flex',
+  },
+  icon: {
+    display: 'flex',
+    alignItems: 'center',
+    ...shorthands.padding(tokens.spacingHorizontalXS),
+  },
+  small: {
+    height: '24px',
+    ...shorthands.padding(tokens.spacingHorizontalSNudge),
+    ...typographyStyles.caption1,
+  },
+  medium: {
+    height: '32px',
+    ...shorthands.padding(tokens.spacingHorizontalSNudge),
+    ...typographyStyles.body1,
+  },
+  large: {
+    height: '40px',
+    ...shorthands.padding(tokens.spacingHorizontalS),
+    ...typographyStyles.body2,
+  },
+  overflow: {
+    ...shorthands.padding(tokens.spacingHorizontalSNudge),
+    '&:hover': {
+      textDecorationLine: 'none',
+    },
+  },
+  currentSmall: {
+    ...typographyStyles.caption1Strong,
+  },
+  currentMedium: {
+    ...typographyStyles.body1Strong,
+  },
+  currentLarge: {
+    ...typographyStyles.subtitle2,
   },
-
-  // TODO add additional classes for different states and/or slots
 });
 
 /**
  * Apply styling to the BreadcrumbLink slots based on the state
  */
 export const useBreadcrumbLinkStyles_unstable = (state: BreadcrumbLinkState): BreadcrumbLinkState => {
   const styles = useStyles();
-  state.root.className = mergeClasses(breadcrumbLinkClassNames.root, styles.root, state.root.className);
 
-  // TODO Add class names to slots, for example:
-  // state.mySlot.className = mergeClasses(styles.mySlot, state.mySlot.className);
+  // const currentSizeMap = {
+  //   small: styles.currentSmall,
+  //   medium: styles.currentMedium,
+  //   large: styles.currentLarge,
+  // };
```

Comment:
Size logic is added in #27428 . Will uncomment it when BreadcrumbButton is merged